docs/analytics.md

```diff
@@ -0,0 +1,22 @@
+# Flank Analytics
+
+Flank makes use of Bugsnag as its only analytics platform. It uses Bugsnag as an error monitor and project stability tool.
+
+For more information about Bugsnag please visit its website [here](https://www.bugsnag.com/) and documentation [here](https://docs.bugsnag.com/)
+
+## Control
+
+It is possible to disable the Bugsnag analytics integration used by flank or to make use of custom Bugsnag key so that any analytics can be displayed for custom use.
+
+ - To disable, place the word ```DISABLED``` in the file ````~/.gsutil/analytics-uuid```` and Flank will no longer send any analytics data
+   - Or simply running the following code (make sure the file exists!) will disable: ```echo "DISABLED" > ~/.gsutil/analytics-uuid```
+
+ - To enable or reenable bugsnag simply remove the file ```~/.gsutil/analytics-uuid``` and the project will default to make use of the flanks Bugsnag key
+
+ - To make use of custom Bugsnag integration, place your bugsnag key/UUID in ```~/.gsutil/analytics-uuid``` and the flank project will make use of it instead of the default key.
```

Comment:
Is this true? I believe we only check the contents of this file for "DISABLED" A user can opt out of bugsnag, however they can't point flank to their own bugsnag instance.

Reply:
It appears that I was mistaken here. There is a test class that works with a provided UUID which, after some investigation, is more important for gclouid than for flank. After another lookthrough the code if the user supplies a UUID it will simply make use of Flanks API key. Only Mock, Disabled will change the behaviour.